torch/csrc/distributed/rpc/init.cpp

```diff
@@ -41,7 +42,26 @@ PyObject* rpc_init(PyObject* /* unused */) {
           [&](FutureMessage& fut) {
             return to_py_obj(fut.wait());
           },
-          py::call_guard<py::gil_scoped_release>());
+          py::call_guard<py::gil_scoped_release>())
+      .def("get",
```

Comment:
Good point on mentioning Java's and Folly's API. This was originally added to address [this comment](https://github.com/pytorch/pytorch/pull/23228#discussion_r309426549) in #23228. But I agree with you that staying consistent with Java and Folly might help to avoid surprising users. I will make the change.